.github/ISSUE_TEMPLATE/chart_upgrade.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Chart Upgrade
+about: Request to change a Chart version
+title: ''
+labels: improvement
+assignees: ''
+
+---
+
+**What Chart should be upgraded?**
+
+/e.g., dexidp/dex 2.15/
```

Comment:
Would it be good to also state why the chart should be upgraded? Mostly so we know if there is some new/changed feature that we should keep in mind while doing the task.
E.g. upgrade chart in order to get access to the new resource request helm variables, or upgrade chart to get latest application version.

Or do we just use these issues when we are upgrading for the sake of staying up to date and use the other issues when we are after a specific feature/change?